.github/workflows/api-integration-tests.yml

```diff
@@ -141,12 +141,3 @@ jobs:
           bats apps/news/tests/command
 
           kill %1
-
-      - name: Prep PHP tests
-        working-directory: ../server/apps/news
-        run: make php-test-dependencies
-
-      - name: Feed tests
```

Comment:
Not sure why, but let's do it

Reply:
I removed these lines because they make no sense to me.

I noticed that the integration tests fail sometimes because of composer timeout, so I thought ok I'm gonna check if the timeout can be extended or something but then it turned out that we run these things and after that nothing happens.

Maybe they were leftovers from previous usage of actions? Because why should we install the php dependencies to then not use them and then check the test feeds but after that we do not do anything with the feeds...

I will look at it again but I'm not sure if we even still use any actual online feed.